code/game/gamemodes/game_mode.dm

```diff
@@ -285,7 +285,6 @@
 
 ///Called by the gameSSticker
 /datum/game_mode/process()
-	return 0
```

Comment:
?

Reply:
The process is called at:
https://github.com/BeeStation/BeeStation-Hornet/blob/1d6bacb2eefe4e7ba97b7ca74d0d39dcbfc50cd8/code/controllers/subsystem/ticker.dm#L205
So I just yeeted returns values away (and also thought about yeeting the proc away since it isn't necessary but I decided against it because how could one figure it out when there is no indication that it is called by SSticker not other processing subsystems; overhead of proc name searching at proc call shouldn't be that great and `process()` really isn't salvageable in that regards anyway.)

Reply:
You shouldn't just leave an empty proc typepath.

Reply:
I think... I should remove this unused proc typepath and leave a comment that some special game_mode `process()`s (they aren't many in the first place) are handled by `SSticker`?